osidb_bindings.spec

```diff
@@ -0,0 +1,58 @@
+%define name osidb_bindings
+%define version 3.6.0
```

Comment:
There is already 3.7.0 version of bindings, do you want to use 3.6.0 as a playground and when things are solid, you will update to 3.7.0. There is a process how OSIDB and OSIDB bindings versions are being synced.

Reply:
Whoops,  I'll add this to file to the 'update_version' function of helpers.sh.